swagger_to_sdk_config.json

```diff
@@ -151,45 +150,32 @@
       ]
     },
     "azure_mgmt_features": {
-      "swagger": "arm-resources/features/2015-12-01/swagger/features.json",
+      "swagger": "specification/resources/resource-manager/readme.md",
       "autorest_options": {
         "Namespace": "Azure::ARM::Features",
         "PackageName": "azure_mgmt_features",
-        "PackageVersion": "0.10.0"
+        "PackageVersion": "0.10.0",
+        "Tag": "package-features-2015-12"
       },
       "output_dir": "management/azure_mgmt_features/lib",
       "wrapper_filesOrDirs": [
         "azure_mgmt_features.rb"
       ]
     },
     "azure_mgmt_graph": {
-      "swagger": "arm-graphrbac/compositeGraphRbacManagementClient.json",
+      "swagger": "specification/graphrbac/data-plane/readme.md",
       "autorest_options": {
         "Namespace": "Azure::ARM::Graph",
         "PackageName": "azure_mgmt_graph",
-        "Modeler": "CompositeSwagger",
         "PackageVersion": "0.10.0"
```

Comment:
Note that autorest_options have changed, and you must use the new one:
- package-version and not packageversion